.circleci/run.sh

```diff
@@ -139,19 +139,19 @@ coverage()
 
     # rebuild dmd with coverage enabled
     # use the just build dmd as host compiler this time
-    local build_path=generated/linux/release/$MODEL
+    local build_path=generated/linux/$BUILD/$MODEL
     # `generated` gets cleaned in the next step, so we create another _generated
     # The nested folder hierarchy is needed to conform to those specified in
     # the generate dmd.conf
     mkdir -p _${build_path}
     cp $build_path/dmd _${build_path}/host_dmd
     cp $build_path/dmd.conf _${build_path}
-    make -j$N -C src -f posix.mak MODEL=$MODEL HOST_DMD=../_${build_path}/host_dmd PIC="$PIC" clean
-    make -j$N -C src -f posix.mak MODEL=$MODEL HOST_DMD=../_${build_path}/host_dmd ENABLE_COVERAGE=1 ENABLE_WARNINGS=1 PIC="$PIC"
```

Comment:
Why remove `ENABLE_WARNINGS`?

Reply:
Reverted this change because it's unrelated (although doesn't matter because  warnings are enabled by default)